stage6/02-adi-update-tools/00-run.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash -e
 
 on_chroot << EOF
-git clone https://github.com/analogdevicesinc/linux_image_ADI-scripts
+[ -d "linux_image_ADI-scripts" ] || {
+	git clone https://github.com/analogdevicesinc/linux_image_ADI-scripts
```

Comment:
should this do a `git fetch/pull` in case it exists?

Reply:
it only happens when i preserve the container for docker when i am testing things. It is more of a change to not break my own builds.

Reply:
The re-runs usually happen after something breaks the build and I restart it after a couple of minutes so there is no need for a git fetch/pull.